### PR TITLE
Rename remote epic test

### PR DESCRIPTION
**Pull request by nicl**
We want to disambiguate results now that we've made various fixes.

cc @tomrf1 is changing the test ID and campaign code enough or do I need to change other stuff too?

### CODE REVIEW

**Review by andre1050 — APPROVED** [review]

File: static/src/javascripts/projects/common/modules/experiments/tests/remote-render-epic.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ const decodeJson = response => response.json();
 const products = ['CONTRIBUTION', 'MEMBERSHIP_SUPPORTER'];
 
 const remoteRenderTest = {
-    id: 'RemoteRenderEpic',
+    id: 'RemoteRenderEpicRoundTwo',
```
Comment (tomrf1) [inline static/src/javascripts/projects/common/modules/experiments/tests/remote-render-epic.js]: do we need to change the switch name as well?

Comment (nicl) [reply static/src/javascripts/projects/common/modules/experiments/tests/remote-render-epic.js]: Yeah sounds sensible. Is that required for the data side, or more about clarity?

Comment (tomrf1) [reply static/src/javascripts/projects/common/modules/experiments/tests/remote-render-epic.js]: possibly the test will not run if it cannot be switched on

**Review by tomrf1 — APPROVED** [review]
thanks

**Pull request merged**